practitioner-service/src/test/java/uk/ac/ox/ndph/mts/practitioner_service/validation/PractitionerValidationTests.java

```diff
@@ -0,0 +1,205 @@
+package uk.ac.ox.ndph.mts.practitioner_service.validation;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.converter.ConvertWith;
+
+import static org.mockito.Mockito.when;
+
+import java.util.LinkedList;
+import java.util.List;
+import org.mockito.Mock;
+import org.junit.jupiter.api.Assertions;
+import uk.ac.ox.ndph.mts.practitioner_service.model.Practitioner;
+import uk.ac.ox.ndph.mts.practitioner_service.model.PractitionerAttributeConfiguration;
+import uk.ac.ox.ndph.mts.practitioner_service.NullableConverter;
+import uk.ac.ox.ndph.mts.practitioner_service.configuration.PractitionerConfigurationProvider;
+import uk.ac.ox.ndph.mts.practitioner_service.model.PractitionerConfiguration;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class PractitionerValidationTests {
+
+    @Mock
+    private PractitionerConfigurationProvider configurationProvider;
+
+    private static List<PractitionerAttributeConfiguration> ALL_REQUIRED_UNDER_35_MAP;
+    static {
```

Comment:
private static final List<PractitionerAttributeConfiguration> ALL_REQUIRED_UNDER_35_MAP = List.of(
        new PractitionerAttributeConfiguration("givenName", "Given Name", "^[a-zA-Z]{1,35}$"),
        new PractitionerAttributeConfiguration("familyName", "Family Name", "^[a-zA-Z]{1,35}$"),
        new PractitionerAttributeConfiguration("prefix", "Prefix", "^[a-zA-Z]{1,35}$"));